### ISSUE TITLE
image alt text improvements

### ISSUE THREAD

**Issue by danielroe** [elk-zone/elk#41]
it should be possible to:
 - [x] set image alt text when uploading images
 - [x] view image alt text by clicking on the image or in a button in an overlay 
 - [x] edit image alt text when editing a post
